src/CarbonAware.LocationSources/src/CarbonAware.LocationSources.csproj

```
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<ItemGroup>
+		<ProjectReference Include="..\..\CarbonAware\src\CarbonAware.csproj" />
+	</ItemGroup>
+
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+		<EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+	</PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="$(ProjectDir)..\..\data\data-files\**\*.*" Link="\location-sources\json\%(RecursiveDir)%(Filename)%(Extension)">
```

Wont this also pull in the test emissions files as well?  Perhaps we should enforce more structure around where files can just be dropped.   EG:

```suggestion
    <Content Include="$(ProjectDir)..\..\data\data-files\location-sources\**\*.*" Link="\location-sources\%(RecursiveDir)%(Filename)%(Extension)">
```

---

I was thinking about this dir structure, and maybe it would be better to have a `..\data\location-files\...`. Having a sub dir under `data-files` is going to do a recursive copy, so basically we are not getting any benefit.

---

Proposed to have `data\locations-sources\**\*.*` and moved only `azure-regions.json` to this new directory.